src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,81 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.exceptions.DukeOnlyIoException;
+
+/**
+ * Storage class that is used for the reading and writing of the list file
+ */
+public class Storage {
+

```

Comment:
Stating class variables as null seems redundant as removing " = null " gives the same effect. Not really a coding violation but lesser code is written as a result.

Reply:
Good point even though this is not really a coding violation